langchain4j-jlama/pom.xml

```diff
@@ -81,17 +82,6 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
-
-        <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-core</artifactId>
```

Comment:
Why removing these? They are required for `JlamaAiServicesWithToolsIT`

Reply:
As Java 21 and 23 builds are green, it means it's safe to removed this dependency here.
It is now transitive from [parent pom](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-parent/pom.xml#L363-L369).